internal/cmd/ske/kubeconfig/login/login.go

```diff
@@ -0,0 +1,240 @@
+package login
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/cache"
+	"k8s.io/client-go/rest"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientauthenticationv1 "k8s.io/client-go/pkg/apis/clientauthentication/v1"
+	"k8s.io/client-go/tools/auth/exec"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	expirationSeconds     = 30 * 60          // 30 min
+	refreshBeforeDuration = 15 * time.Minute // 15 min
+)
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "login",
+		Short: "Login plugin for kubernetes clients",
+		Long: fmt.Sprintf("%s\n%s",
+			"Login plugin for kubernetes clients, that creates short-lived credentials to authenticate against a STACKIT Kubernetes Engine (SKE) cluster.",
+			"To obtain a kubeconfig for use with the login command, run 'kubeconfig create --login'.",
+		),
+		Args: args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Get a login kubeconfig for the SKE cluster with name "my-cluster". `+
+					"This kubeconfig does not contain any credentials and instead obtains valid credentials via the STACKIT CLI.",
+				"$ stackit ske kubeconfig create my-cluster --login"),
+			examples.NewExample(
```

Comment:
these two examples should be merged, they are part of the same flow

Reply:
How can I merge them? There is only one flow in general. I don't think that separate examples mean different flows...

Reply:
Yeah in that case it's ok like this. I meant that usually we have different examples for different use cases but here they are the same